package.json

```diff
@@ -5,7 +5,8 @@
   "type": "module",
   "scripts": {
     "dev": "vite",
-    "build": "tsc && vite build",
+    "build": "tsc && npx cross-env VITE_TABLET=768 VITE_MOBILE=425 vite build && npm run optimizeImages",
+    "optimizeImages": "npx cross-env VITE_TABLET=768 VITE_MOBILE=425 VITE_COMPRESS_QUALITY=80 node optimizeImages/optimizeImages.js",
```

Comment:
Is there a real need for duplicating env variables in both scripts?

Reply:
Unfortinally, yes. Defining the env variables this way - defines for the each script's runtime separatelly.

for example i have tried to defining variables like this, but variables will be only specified for ```tsc``` runtime and not for the other ones
```
npx cross-env VITE_TABLET=768 VITE_MOBILE=425 VITE_COMPRESS_QUALITY=80 tsc && vite build && npm run optimizeImages
```

Unless i missing something